bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -229,6 +229,14 @@ public String toString() {
             var m = BACKPORT_TITLE_PATTERN.matcher(pr.title());
             if (m.matches()) {
                 var hash = new Hash(m.group(1));
+                if (pr.headHash().equals(hash)) {
```

Comment:
Isn't it the hash of the first commit in the PR, rather than the HEAD commit of the PR, that is likely to cause the problem? In case there is more than one commit in the PR, might it be better to check whether it is an ancestor of the HEAD commit?

Reply:
Right, if there are multiple changes in the PR branch, any one of them could potentially be the original change. Checking that is a bit more complicated though as I think it requires a local clone to inspect. I will need to investigate how best to solve it.